### ISSUE TITLE
Focused block border color contrast ratio too low

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#7439]
When tabbing through the blocks using the keyboard, I can barely see which of the blocks is the focused one. I have some eyesight problem 🙂and I'm not the only one. There are a lot of users with vision impairments who would benefit from a higher contrast ratio.

The low contrast of the border around the focused block is particularly evident when the formatting toolbar is fixed to the top, see screenshot below (hint: the focused block is the second paragraph):

![screen shot 2018-06-21 at 16 28 38](https://user-images.githubusercontent.com/1682452/41725731-ce90b962-7570-11e8-8a94-f0b9eb9fe7cb.png)

Instead, when the formatting toolbar is attached to the block, the presence of the UI helps a bit but it doesn't fully solve the issue.

I'd propose to consider to increase the gray border contrast. I guess it's not a big visual change, as the contrast when hovering is already way higher. Worth reminding the perceived contrast is not about the shade of color but it's about the luminosity contrast, and the blue used on hover has a way higher luminosity contrast: 
 
![screen shot 2018-06-21 at 16 28 13](https://user-images.githubusercontent.com/1682452/41726046-967e4642-7571-11e8-9a40-f6ca069a6006.png)

Aside:
I'm not fully convinced about the blue, as the convention in WordPress is to use it for actionable controls; regardless, whether the final color will be blue or gray, the luminosity contrast should be higher.

/Cc @jasmussen @karmatosed

**Comment by jasmussen** [comment]
Can you take a look at the navigation mode solution propose din https://github.com/WordPress/gutenberg/pull/5709#issuecomment-393791220? In that design, navigating through blocks would show the blue outline.

**Comment by afercia** [comment]
The blue outline (or any other color with a higher luminosity contrast ratio) would be beneficial. It should be blue also when clicking on the block though.

Re: concerns with the blue which is a convention in WordPress: are we sure we want to use a blue?

**Comment by karmatosed** [comment]
I feel as we use blue throughout WordPress using it here is good. Assuming #5709 gets in what is the lightest but still accessible color of grey we can use here @afercia? Let's use that as a starting point.

**Comment by afercia** [comment]
Fine, as long as the contrast ratio is at least 3:1 (non-text contrast). Noting this is basically the same issue pointed out in https://github.com/WordPress/gutenberg/issues/7053 /Cc @anevins12

**Comment by nbrummerstedt** [comment]
A general setting for optional high contrast ratio would be beneficial.

**Comment by karmatosed** [comment]
@nbrummerstedt are you suggesting an in general setting for WordPress? I ask as know it can adapt to high contrast settings for example on Windows.

**Comment by nbrummerstedt** [comment]
Come to think of it, such a setting should obviously be a site-wide user-setting. In that case, the suggestion belongs to wordpress/core ? 
The reason I suggest it is because light grays and low contrasts can be really valuable to reduce noise and additionally hint functionality for users who are not visually impaired.

**Comment by karmatosed** [comment]
@nbrummerstedt that could be possible, core would be the place to suggest such a thing yes. I do wonder though if someone that wanted it would have already set that setting on their computer, that maybe is an assumption.

**Comment by nbrummerstedt** [comment]
@karmatosed As I see it, such a setting should be set on a per user basis, and be disabled by default.

A high contrast stylesheet should be enqueued if any of the following :
- User has manually enabled the setting
- User has enabled High Contrast in OS
- User has enabled High Contrast in browser (if such an option exists)

**Comment by afercia** [comment]
Worth reminding the color contrast ratio should meet the requirement by default, pretty much as the default WordPress admin color scheme aims to do.

**Comment by rahulbroy** [comment]
@karmatosed Blue should be fine I think, as long as the demarcation is clear.
I would also suggest to look into the contrast of the editor demarcation in general. While using the standard settings, everything is in white and I can't distinguish the difference between the background and the editor. One can get used to it but its still undesirable.

**Comment by karmatosed** [comment]
I think I'm right in closing this as we have #5709 suggesting a solution? If I am wrong we can reopen.
